clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/security/AppengineNamedAccountCredentials.groovy

```diff
@@ -54,6 +54,11 @@ class AppengineNamedAccountCredentials implements AccountCredentials<AppengineCr
   final GcloudReleaseTrack gcloudReleaseTrack
   final List<AppengineGitCredentialType> supportedGitCredentialTypes
 
+  final List<String> services
+  final List<String> versions
+  final List<String> omitServices
+  final List<String> omitVersions
```

Comment:
I would put a check somewhere that at most one of `X`, and `omitX` is specified.

Reply:
Maybe it's useful now that regex is supported -- without it I think it's unnecessary.

Reply:
Oh wait, do you mean here that I should check if any string appears both in `X` and `omitX`?  That would indeed be wrong.